README.md

```diff
@@ -271,7 +253,68 @@ the [Maven version range format](https://cwiki.apache.org/confluence/display/MAV
 | (,1.0],[1.2,) | x <= 1.0 or x >= 1.2. Multiple sets are comma-separated                       |
 | (,1.1),(1.1,) | This excludes 1.1 if it is known not to work in combination with this library |
 
-#### External Dependencies: Runs
+#### Local Files
+
+You can also include files built by other tasks in your project, for example, jar tasks of other source sets.
+
+When wanting to build a secondary jar for a coremod or service, you could define a separate source set `service`,
+add a jar task to package it and then include the output of that jar like this:
+
+```groovy
+sourceSets {
+    service
+}
+
+
+neoForge {
+    // ...
+    mods {
+        // ...
+        // To make the service load in dev
+        'service' {
+            sourceSet sourceSets.service
+        }
+    }
+}
+
+def serviceJar = tasks.register("serviceJar", Jar) {
+    from(sourceSets.service.output)
+    archiveClassifier = "service"
+    manifest {
+        attributes(
+                'FMLModType': "LIBRARY",
```

Comment:
If it really is a "service" that we are talking about, then this is not necessary!

Reply:
If it's a library, it goes onto the PLUGIN layer.
If it should go on the SERVICE layer then it needs to implement one of the services.
It's fine to have a plugin, but I wouldn't call it "service" in that case.